.github/workflows/stage-1-commit.yaml

```diff
@@ -86,6 +94,27 @@ jobs:
         uses: actions/checkout@v4
       - name: "Lint Terraform"
         uses: ./.github/actions/lint-terraform
+  terraform-testing:
+    name: "Terraform Testing"
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    environment: dev
```

Comment:
It's hurting my brain and sensibilities that this for some reason still seems to need to be dev not main. It'll make sense somehow I'm sure.

Reply:
the environment here is what github environment relates to, so if you set one called main in there and set the secrets, I think it can be changed in here

Reply:
aha of course!